website/cue/reference/releases/0.43.0.cue

```diff
@@ -0,0 +1,449 @@
+package metadata
+
+releases: "0.43.0": {
+	date:     "2024-11-27"
+	codename: ""
+
+	whats_next: []
+
+	description: """
+		The Vector team is pleased to announce version 0.43.0!
+		This release contains a number of new components such as the `OpenTelemetry` sink and the `ExclusiveRoute` sink.
+		It also includes numerous enhancements and fixes as listed below.
+		There are no breaking changes or deprecations with this release and so no upgrade guide.
+		"""
+
+	changelog: [
+		{
+			type: "feat"
+			description: """
+VRL was updated to v0.20.0. This includes the following changes:
+
+### Breaking Changes & Upgrade Guide
+
+- Fixes the `to_float` function to return an error instead of `f64::INFINITY` when parsing [non-normal](https://doc.rust-lang.org/std/primitive.f64.html#method.is_normal) numbers. (https://github.com/vectordotdev/vrl/pull/1107)
+
+### New Features
+
+- The `decrypt` and `encrypt` VRL functions now support aes-siv (RFC 5297) encryption and decryption. (https://github.com/vectordotdev/vrl/pull/1100)
+
+### Enhancements
+
+- `decode_punycode` and `encode_punycode` with `validate` flag set to false should be faster now, in cases when input data needs no encoding or decoding. (https://github.com/vectordotdev/vrl/pull/1104)
+- `vrl::value::Value` now implements `PartialCmp` that first checks whether the enum discriminants
+	(that both are floats for example), and if they are calls `partial_cmp` on the inner values.
```

Comment:
Can you explain what this sentence means? Especially "checks whether the enum discriminants (that both are floats for example)"

Reply:
`Value` is an enum, `PartialCmp` tries to compare two `Value`s but those can actually hold a value of different type. We could actually shorten this to 

```
`vrl::value::Value` now implements `PartialCmp`
```

Reply:
```
... now implements `PartialOrd`
```
correct?

Reply:
This is the PR: https://github.com/vectordotdev/vrl/pull/1117/files. It basically added a trait implementation. On a second thought, this affect developers and not users. Maybe it is too low level for this changelog?

Reply:
Yeah, and technically it's a VRL implementation change that also isn't used directly by end users.  Ok to exclude.

Reply:
Removed in https://github.com/vectordotdev/vector/pull/21906/commits/5ff6334d61e5a151fe1fa375bd2a8fb68f3de950.